src/D2L.CodeStyle.Analyzers/ApiUsage/Logging/LoggingExecutionContextScopeBuilderAnalyzer.cs

```diff
@@ -184,7 +183,7 @@ CancellationToken ct
 		}
 
 		private static bool IsAwaitable(
-			IImmutableSet<INamedTypeSymbol> taskTypeBuiltins,
+			IImmutableSet<ISymbol> taskTypeBuiltins,
 			INamedTypeSymbol AsyncMethodBuilderAttribute,
 			ITypeSymbol type
 		) {

```

Comment:
Down below it was passing `ITypeSymbol` into `.Contains` which causes Linq to take over:

![image](https://user-images.githubusercontent.com/12101120/151647359-2b57c424-7850-4e1e-ae5f-413d7260c402.png)